std/traits.d

```diff
@@ -6845,8 +6845,9 @@ template getSymbolsByUDA(alias symbol, alias attribute) {
     }
 
     enum hasSpecificUDA(string name) = mixin("hasUDA!(symbol.%s, attribute)".format(name));
-
-    alias membersWithUDA = toSymbols!(Filter!(hasSpecificUDA, __traits(allMembers, symbol)));
+    enum noThisMember(string name) = (name != "this");
```

Comment:
Please add a comment with a short description (filtering out nested class context) or choose a more descriptive name for the filter. Also please refer to the Bugzilla Ticket.